docs/DORA.md

```diff
@@ -164,27 +150,42 @@ workflows:
       - deploy
 ```
 
-If you have set a [username/password](GettingStarted/Authentication.md) for Config UI, you need to add them to the curl to register a deployment:
 
-```
-curl https://sample-url.com/api/plugins/webhook/2/deployments -X 'POST' -u 'username:password' -d '{
-    \"commit_sha\":\"$CIRCLE_SHA1\",
-    \"repo_url\":\"$CIRCLE_REPOSITORY_URL\",
-    \"start_time\":\"$start_time\"
-  }'
-```
+2.5 Run the CircleCI pipeline. Visit DevLake's DB to check if the deployments have been successfully pushed to DevLake. The deployments will appear in table [cicd_deployments](DataModels/DevLakeDomainLayerSchema.md#cicd_deployments) and [cicd_deployment_commits](DataModels/DevLakeDomainLayerSchema.md#cicd_deployment_commits) in DevLake's database.
+
+  ![CircleCI-screenshot](https://i.imgur.com/IyneAMn.png)
```

Comment:
Is it a mistake ? I think we shouldn's depend on imgur.com.

Reply:
@d4x1 We shouldn't rely on imgur.com. I'll get them updated.